### PR TITLE
Fix issue #129

### PR DESCRIPTION
**Pull request by ayman-alkom**
Fix issue #129 
I will rebase it tommorow

### CODE REVIEW

File: src/View/UIViewTrait.php [hunk 1]
```diff
@@ -13,11 +13,16 @@
...
+     * - `layout` to pass your layout, or it will use the plugin one
+     *
+     * @param array $options
      * @return void
      */
-    public function initializeUI()
+    public function initializeUI(array $options = [])
     {
-        if (empty($this->layout) || $this->layout === 'default') {
+        if (empty($options['layout'])) {
```
Comment (jadb) [inline src/View/UIViewTrait.php]: I think better to allow one to pass `layout = false` or `layout = ...`  if they don't want the default one or if they want a custom one instead. If `TRUE` is passed or option is not defined, use plugin's default.

Right now, `FALSE` will still define the plugin's layout and `TRUE` will just do nothing.

Comment (ayman-alkom) [inline src/View/UIViewTrait.php]: yes @jadb I thought like you, but I didn't found it meaningful. take another look, I may not right

Comment (jadb) [inline src/View/UIViewTrait.php]: Tried using it, I think the suggestion does make sense.

**Comment by ayman-alkom** [comment]
Sorry @jadb the PR has been closed while i was trying to rebase it

**Pull request closed without merging**